### DISCUSSION TITLE
Still receiving Queue polling for deleted Azure Function

### DISCUSSION THREAD

**Discussion by gorillapower** [Azure/azure-functions-durable-extension#2468] [Q&A] [answered]
We seem to be receiving queue activity for a function that has been deleted in Azure. 

We added logging on our storage account to see what the source of the traffic could be, but all I was able to find was an IP address that doesnt point to any provisioned appservice in our subscription.

The only reference to this deleted azure function was a fileshare that I found in the storage account, I thought deleting it would solve these phantom queue polling, but its still happening. 

I also learnt that the queue polling can happen from the **Scale Controller** and the **Function Host**. 

Is the Scale Controller polling my queue even though the Function host is gone?

I can provide details if need be.

**Comment by davidmrdavid** [comment]
Hi @gorillapower,

Apologies for the late reply. If the app / function is deleted, then the source is probably something else.
You're right that the Scale Controller monitors storage resources, especially when those control scaling.

Two quick questions:
(1) What SKU/Plan is this behavior occurring on?
(2) What storage resources is being polled? A control queue, a work item queue, a fileshare? I wasn't entirely clear on this.

Thanks!

**Comment by gorillapower** [comment, marked as answer]
Hi @davidmrdavid 

No worries, thank you! 

After you mentioned which queue was being polled, I took another look and noticed its not a Durable Function queue! Its another custom queue that we created...however the mystery continues as to what is polling it.

Thanks for your attention
